packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -4069,6 +4069,58 @@ void main() {
     expect(tester.getTopLeft(find.text('label')).dy, 20.0);
   });
 
+  testWidgets('InputDecorator floating label width scales when focused', (WidgetTester tester) async {
+    final String longStringA = String.fromCharCodes(List<int>.generate(200, (_) => 65));
+    final String longStringB = String.fromCharCodes(List<int>.generate(200, (_) => 66));
+
+    await tester.pumpWidget(
+      Center(
+        child: Container(
+          width: 100,
+          height: 100,
+          child: buildInputDecorator(
+            // isFocused: false (default)
```

Comment:
remove this

Reply:
It is to make it more clear and to be consistent with the other tests in this file